CHANGELOG.md

```diff
@@ -4,6 +4,57 @@
 
 From v1.2, Pantheon will require Java 11.  That is, Pantheon on Java 8 will no longer be supported.
 
+## 1.1.3
+
+### Additions and Improvements
+
+- Added liveness and readiness probe stub endpoints [\#1553](https://github.com/PegaSysEng/pantheon/pull/1553) 
+- Implemented operator tool. \(blockchain network configuration for permissioned networks\) [\#1511](https://github.com/PegaSysEng/pantheon/pull/1511) 
+- \[PAN-2754\] Added eea\_getPrivacyPrecompileAddress [\#1579](https://github.com/PegaSysEng/pantheon/pull/1579) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Publish the chain head gas used, gas limit, transaction count and ommer metrics [\#1551](https://github.com/PegaSysEng/pantheon/pull/1551)
+- Add subscribe and unsubscribe count metrics [\#1541](https://github.com/PegaSysEng/pantheon/pull/1541)
+- Add pivot block metrics [\#1537](https://github.com/PegaSysEng/pantheon/pull/1537)
+
+Documentation updates include: 
+
+- Added [debug\_traceBlock\* methods](https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-API-Methods/#debug_traceblock) 
+- Reorganised [monitoring documentation](https://docs.pantheon.pegasys.tech/en/latest/Monitoring/Monitoring-Performance/)
+- Added [link to sample Grafana dashboard](https://docs.pantheon.pegasys.tech/en/latest/Monitoring/Monitoring-Performance/#monitor-node-performance-using-prometheus) 
+- Added [note about replacing transactions in transaction pool](https://docs.pantheon.pegasys.tech/en/latest/Using-Pantheon/Transactions/Transaction-Pool/#replacing-transactions-with-same-nonce)
+- Updated [example transaction scripts](https://docs.pantheon.pegasys.tech/en/latest/Using-Pantheon/Transactions/Transactions/#example-javascript-scripts)
+- Updated [Alethio Ethstats and Explorer documentation](https://docs.pantheon.pegasys.tech/en/latest/Monitoring/Alethio/Overview/)
+
+### Technical Improvements 
+
+- \[PAN-2811\] Be more lenient with discovery message deserialization [\#1580](https://github.com/PegaSysEng/pantheon/pull/1580) 
```

Comment:
This is actually an important bug fix which enables Pantheon to work on Rinkeby again.  It completes our support for EIP-8.

Reply:
Fixed